Jenkinsfile

```diff
@@ -124,161 +125,175 @@ def sendNotifications(String buildStatus = '', String stageName = '', String git
 
 // begin pipeline
 try {
-	stage('Prepare') {
-		currentStage = "${env.STAGE_NAME}"
-		node {
-			// Grab the commit that triggered the build.
-			checkout scm
-
-			// Load the child Jenkinsfiles.
-			scriptForApps = load('apps/build.groovy')
-			scriptForDeploys = load('ops/deploy-ccs.groovy')
+	podTemplate(containers: [containerTemplate(name: 'bfd-cbc-build', image: 'public.ecr.aws/c2o1d8s9/bfd-cbc-build:jdk8-an29-tf12', command: 'cat', ttyEnabled: true, alwaysPullImage: true, resourceLimitCpu: '4000m', resourceLimitMemory: '8192Mi', resourceRequestCpu: '4000m', resourceRequestMemory: '8192Mi')], serviceAccount: 'bfd') {
+		node(POD_LABEL) {
+			stage('Prepare') {
+				currentStage = "${env.STAGE_NAME}"
+				container('bfd-cbc-build') {
+					// Grab the commit that triggered the build.
+					checkout scm
+
+					// Load the child Jenkinsfiles.
+					scriptForApps = load('apps/build.groovy')
+					scriptForDeploys = load('ops/deploy-ccs.groovy')
+
+					// Set global AWS environment variables from role assumption
+					withCredentials([string(credentialsId: 'bfd-aws-assume-role', variable: 'awsAssumeRole')]) {
+						awsCredentials = sh(returnStdout: true, script: "aws sts assume-role --role-arn ${awsAssumeRole} --role-session-name session --output text --query Credentials").trim().split(/\s+/)
+						env.AWS_DEFAULT_REGION = 'us-east-1'
+						env.AWS_ACCESS_KEY_ID = awsCredentials[0]
+						env.AWS_SECRET_ACCESS_KEY = awsCredentials[2]
+						env.AWS_SESSION_TOKEN = awsCredentials[3]
+					}
```

Comment:
Here's where we assume the role in the BFD account.  This gives us a unique set of credentials on every job run.